src/framework/mlt_log.h

```diff
@@ -90,4 +91,16 @@ int mlt_log_get_level( void );
 void mlt_log_set_level( int );
 void mlt_log_set_callback( void (*)( void*, int, const char*, va_list ) );
 
+#define mlt_log_timings_begin() \
+{ \
+	int64_t _mlt_log_timings_begin = mlt_log_timings_now(), _mlt_log_timings_end;
+
+#define mlt_log_timings_end(service, msg) \
+	_mlt_log_timings_end = mlt_log_timings_now(); \
+	mlt_log_verbose( service, "%s:%d: T(%s)=%" PRId64 " us\n", \
```

Comment:
A lot of users or possibly even apps like Kdenlive or Flowblade use verbose log level, and I am not sure we want this volume. OTOH debug level is so verbose it requires text filtering. Maybe add a new level like MLT_LOG_TIMINGS?